### DISCUSSION TITLE
final verdict

### DISCUSSION THREAD

**Discussion by WhiteCloud0106** [Lost-MSth/Arcaea-server#141] [Q&A] [answered]
我想問那個final包的額外文件（影片檔和影片的音檔）是要放在哪裡？是跟該曲目的譜面和音檔放在一起嗎？
順便問一下axiom of the End 的使用上有沒有什麼限制

**Comment by Lost-MSth** [comment, marked as answer]
视频和谱面以及音频放一起，命名为 `video.mp4` 和 `video_audio.ogg`，有一首非常特别的曲子（我记不得哪个了）的视频文件命名是 `video_720.mp4` 和 `video_1080.mp4`

按理说没有限制，但可能有 bug，无论是客户端问题还是服务端问题都是有可能的

**Reply by WhiteCloud0106** [reply]
之前有遇過外置殼子把東西放好之後，點擊axiom of the End跳出叫我下載東西，但點擊下載之後又報錯說已經有這些東西，然後再點axiom又叫我下載東西的😔
不知道是殼子還是伺服器問題

**Reply by Lost-MSth** [reply]
外置壳？那大概是壳子问题了，检测不到文件就让你下载

或者也有可能是服务端确实缺了某些文件

**Comment by WhiteCloud0106** [comment]


**Reply by Lost-MSth** [reply]
这个我不是很清楚，理论上仿照官方的就行

**Comment by WhiteCloud0106** [comment]
想問一下關於FV包幾個影片和音檔的部分，會需要特殊編碼嗎？
之前拆出來看的時候，影片貌似有特殊編碼，如果跟音檔放在一起會自動抓音檔同時播放。不確定有沒有強制要求要這樣編碼？
再來就是秒數的部分，如果要換成別的影片和音檔的話，秒數要一模一樣嗎？（感覺有點多餘的問題但還是問一下（（

**Reply by Lost-MSth** [reply]
这些问题我完全不知道，理论上不需要特殊编码、秒数也不需要完全一样，但不好说，建议慢慢试（）

**Comment by WhiteCloud0106** [comment]
出了點問題是跑完Testify流程之後並沒有解鎖Testify 有跳出宿命光的畫面還有提示 但實際上都沒有
![Screenshot_20260121_000853](https://github.com/user-attachments/assets/7847bdbe-e466-4453-9da7-bb00156da9af)
![Screenshot_20260121_000900](https://github.com/user-attachments/assets/94178ed9-098a-443a-9472-b3d8421a72d8)
不知道這是伺服器還是客戶端的bug 用的是446的客戶端和2.11.3的伺服器
不過有甚麼方式能強制解鎖嗎 例如伺服器能夠強制客戶端讀取解鎖之類的

**Reply by Lost-MSth** [reply]
服务端在 v2.11.3.12 才还原此功能。

https://github.com/Lost-MSth/Arcaea-server/commit/9636cfcae8f6b38fad32a209a3ffd739bd86dc66

**Reply by WhiteCloud0106** [reply]
啊好的 那我更新一下（
